packages/moonstone/Scrollable/ScrollableNative.js

```diff
@@ -130,6 +127,15 @@ class ScrollableBaseNative extends Component {
 		 */
 		focusableScrollbar: PropTypes.bool,
 
+		/**
+		 * Prevents scroll by dragging or flicking on the list or the scroller.
```

Comment:
How about `scrolling`?

Reply:
`scroll` can be used as a noun and pretty clear here what it means, IMO.